plugins/commands/updateTrusted.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const baseSchema = schema.models.command.base;
+
+module.exports = () => ({
+    method: 'PUT',
+    path: '/commands/{namespace}/{name}/trusted',
+    config: {
+        description: "Update a command's trusted property",
+        notes: 'Returns null if successful',
+        tags: ['api', 'commands', 'trusted'],
+        auth: {
+            strategies: ['token'],
+            scope: ['admin', '!guest']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: async (request, reply) => {
+            const { name, namespace } = request.params;
+            const { commandFactory } = request.server.app;
+            const { trusted } = request.payload;
+
+            const commands = await commandFactory.list({ params: { namespace, name } });
+
+            if (commands.length === 0) {
+                throw boom.notFound(`Command ${namespace}/${name} does not exist`);
+            }
+
+            return Promise.all(
+                commands.map((command) => {
```

Comment:
If I'm reading this correctly, looks like we plan to get **all** the commands with the name/namespace and we're going to go and set each command with the `trusted` value?
If we're checking the MAX value with datastore-sequelize, I think we can just set one of the command versions, right?

Reply:
but that mean u will need to find that one version when u gonna remove it which i rarely think that would ever happen once something got trusted

Reply:
Well after calling this endpoint, any newly published command versions will not have `trusted: true`, anyways, right?

And for removing it, are we trying to use the same endpoint? Or will we make a separate one? Or just rely on a DB call?

I think it should be pretty easy to find the one version. We never delete commands/templates. You can pass in another param to `factory.list` to find it or construct a pretty simple SQL command I think.

Reply:
ok, since u spotted my laziness, i guess i surrender….